lib/Elastica/ResultSet.php

```diff
@@ -14,117 +14,45 @@
 class ResultSet implements \Iterator, \Countable, \ArrayAccess
 {
     /**
-     * Class for the static create method to use.
-     *
-     * @var string
-     */
-    protected static $_class = 'Elastica\\ResultSet';
-
-    /**
      * Results.
      *
-     * @var array Results
+     * @var Result[] Results
      */
-    protected $_results = array();
+    private $_results = array();
```

Comment:
Habit mostly, I dont like protected unless there is a very specific reason for it to be extendable. It makes code more maintainable.